File: scala-pr/54/scala/code-quality-naming/DataModel.scala
class DataModel {
  // {fact rule=code-quality-naming@v1.0 defects=1}
  private var d: List[String] = List.empty
  private var c: Int = 0
  private var f: Boolean = false

  def p(arr: List[String]): Unit = {
    d = List.empty
    c = 0
    f = false

    arr.foreach { i =>
      if (i.nonEmpty) {
        d = d :+ i
        c += 1
      }
    }

    f = c > 0
  }
  // {/fact}
}

File: php-pr/51/php/code-quality-naming/DataModel.php
<?php

class DataModel
{
    // {fact rule=code-quality-naming@v1.0 defects=1}
    private $d;
    private $c;
    private $f;

    public function p($arr)
    {
        $this->d = [];
        $this->c = 0;
        $this->f = false;

        foreach ($arr as $i) {
            if (!empty($i)) {
                $this->d[] = $i;
                $this->c++;
            }
        }

        $this->f = $this->c > 0;
    }
    // {/fact}
}